Config/DefaultEngine.ini

```diff
@@ -1,10 +1,11 @@
 
 
 [/Script/EngineSettings.GameMapsSettings]
-GameDefaultMap=/Game/StarterContent/Maps/Minimal_Default
```

Comment:
the only conflict here is that we used different game modes and starting maps. this is the mapconflict